app/src/main/java/com/simplecity/amp_library/ui/screens/main/LibraryController.java

```diff
@@ -148,12 +148,6 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
 
         setupViewPager();
 
-        compositeDisposable.add(Aesthetic.get(getContext())
```

Comment:
`compositeDisposable.clear()` is still called later in the lifecycle, and other than that is completely unused. Could we remove entirely?